nncf/torch/model_graph_manager.py

```diff
@@ -337,3 +337,28 @@ def get_fake_quantizer(
             if isinstance(module, BaseQuantizer):
                 return module
     return None
+
+
+def get_target_dim_for_weight_compression(metatype: om.PTOperatorMetatype, ndims: int, input_port_id: int) -> int:
```

Comment:
```suggestion
def get_weight_channel_axes(metatype: om.PTOperatorMetatype, ndims: int, input_port_id: int) -> int:
```

Related PR: https://github.com/openvinotoolkit/nncf/pull/2082

Reply:
done